rnaseq/workflow.cwl

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: Workflow
+label: "RnaSeq alignment workflow"
+requirements:
+    - class: MultipleInputFeatureRequirement
+    - class: ScatterFeatureRequirement
+    - class: SubworkflowFeatureRequirement
+inputs:
```

Comment:
There are two relevant options:

```
--rg-id <text>
Set the read group ID to <text>. This causes the SAM @RG header line to be printed, with <text> as the value associated with the ID: tag. It also causes the RG:Z: extra field to be attached to each SAM output record, with value set to <text>.

--rg <text>
Add <text> (usually of the form TAG:VAL, e.g. SM:Pool1) as a field on the @RG header line. Note: in order for the @RG line to appear, --rg-id must also be specified. This is because the ID tag is required by the SAM Spec. Specify --rg multiple times to set multiple fields. See the SAM Spec for details about what fields are legal.
```